core/tasks.py

```diff
@@ -12,40 +13,49 @@
 from chronam.core.models import Batch, OcrDump
 from chronam.core.batch_loader import BatchLoader
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
+@task
+def load_page_in_cache(url):
+    try:
+        _logger.debug("Requesting page [%s]" % url)
```

Comment:
Same comment re: string interpolation. Additionally in this case I would question whether this is necessary given that requests uses urllib3 which already logs the request